tests/metrics/test_github.py

```diff
@@ -0,0 +1,168 @@
+from app.source.github import Github
+from app.daos.dao_git_metric import dao_upsert_git_metric
+from github3.exceptions import NotFoundError
+import datetime
+
+
+def mock_github_client(mocker, no_prs=False, create_date=datetime.datetime(2018, 12, 14)):
+    class MockGithubClient:
+        def __init__(self, token=''):
+            self.token = token
+
+        def search_repositories(self, query):
+            return [Repository()]
+
+    class Repository:
+        def __init__(self):
+            self.repository = Repo()
+            pass
+
+    class Repo:
+        url = "https://github.com/alphagov/test_repo_name"
+        def __init__(self, arg=1):
+            self.name = "test_repo_name"
+
+        def pull_request(self, number):
+            return PullReq()
+
+        def pull_requests(self, state):
+            return PullReqs()
+
+        def commit(self, sha):
+            return RepoCommit()
+
+    class RepoCommit:
+        files = [{
```

Comment:
Looks like the mock data is hard coded, would be better if the mock data can be passed in to test different scenarios.